coffee-bom/pom.xml

```diff
@@ -401,14 +411,20 @@
 				<artifactId>hibernate-validator</artifactId>
 				<version>${version.org.hibernate.validator}</version>
 				<scope>test</scope>
+				<exclusions>
+					<exclusion>
+						<groupId>javax.validation</groupId>
```

Comment:
Ez miert kellett bele? Ahogy neztem nincs is ilyen dependencyje es amugy is csak a test scope-ban van hasznalva

Reply:
ezt az exclusiont vegyuk lecci ki, mivel test scope eleve a bekotes, nincs hatassal a release-re